VnE/Source/MainWindow.py

```
@@ -298,6 +297,7 @@ def __init__(self, parent=None):
         super().__init__(parent=parent)
         self.setWindowIcon(QtGui.QIcon('Source/ico.svg'))
         self.setWindowTitle('Uniforms')
+        self.setWindowFlag(QtCore.Qt.WindowStaysOnTopHint, True)
```

Ruff (0.15.2)

[error] Boolean positional value in function call

(FBT003)